src/api/db/migrate/20170306084558_change_repositories_remote_project_name_to_not_null.rb

```diff
@@ -1,9 +1,13 @@
 class ChangeRepositoriesRemoteProjectNameToNotNull < ActiveRecord::Migration[5.0]
   def up
-    execute 'UPDATE repositories SET remote_project_name = "" WHERE remote_project_name is null'
+    transaction do
+      execute 'UPDATE repositories SET remote_project_name = "" WHERE remote_project_name is null'
+      # drop existsing double entries, must be a ruby loop for the path_elements and reference cleanup
+      Repository.find_by_sql("select A.* from repositories as A  LEFT JOIN repositories as B ON A.db_project_id = B.db_project_id and A.name = B.name where A.id != B.id").each.destroy
```

Comment:
I mean won't this line fail if you have two repositories which have the name and the same db_project_id because of this [unique constraint](https://github.com/evanrolfe/open-build-service/blob/master/src/api/db/structure.sql#L885)?